internal/db/schema/migrations/postgres/13/01_oidc_add_claims.up.sql

```diff
@@ -0,0 +1,8 @@
+begin;
+
+alter table auth_oidc_account
+  add column token_claims text;
```

Comment:
Is an empty string allowed? If not, can a check be added?

Reply:
Originally when I wrote this I thought empty string would be allowed; however it turns out in the code a nil map is not allowed so marshaling empty claims would actually turn into `{}`. So I can put a check in, yes.